src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -24,6 +24,7 @@ public static void MapBackground(IImageHandler handler, IImage image)
 			handler.UpdateValue(nameof(IViewHandler.ContainerView));
 
 			handler.ToPlatform().UpdateBackground(image);
+			handler.ToPlatform().UpdateOpacity(image);
```

Comment:
In Android the Image the ContainerView setting the Background property because is the way to support correctly gradients. In case we set the ContainerView, update the opacity to apply it also to the container.